lib/util/quoteMath.ts

```diff
@@ -0,0 +1,12 @@
+import { BigNumber } from "ethers";
+import { BigNumber as BN } from "bignumber.js";
+import { ClassicQuoteDataJSON, Quote } from "../entities";
+
+// quoteSizeUSD = (multiple of gas-cost-equivalent quote token) * (gas cost in USD)
+export function getQuoteSizeEstimateUSD(classicQuote: Quote) {
+    const classicQuoteData = classicQuote.toJSON() as ClassicQuoteDataJSON;
+  return new BN(
+    BigNumber.from(classicQuoteData.quoteGasAdjusted)
+      .div(BigNumber.from(classicQuoteData.gasUseEstimateQuote)).toString())
+    .times(new BN(classicQuoteData.gasUseEstimateUSD));
```

Comment:
to avoid bignumber.js you could also do `ethers.utils.parseUnits(gasUseEstimateUSD, 6)` then divide out 10**6

Reply:
don't think this comes from parsing USDC with decimals, saw some with 18 decimals